Source/Core/DolphinQt/QtUtils/ScreenSaverInhibitor.cpp

```diff
@@ -0,0 +1,124 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt/QtUtils/ScreenSaverInhibitor.h"
+#include "Common/Logging/Log.h"
+#include "Core/ConfigManager.h"
+
+#if defined(_WIN32)
+#include <Windows.h>
+#elif !defined(__APPLE__)
+#include <QtDBus/QtDBus>
+static const QString s_screen_saver_path = QStringLiteral("/org/freedesktop/ScreenSaver");
```

Comment:
These strings should be `constexpr char` to avoid unnecessary global static constructors.

Reply:
The conversion cost is extremely unlikely to matter in this case. I wouldn't worry about it.